ui/src/views/People/Edit.vue

```diff
@@ -98,70 +64,20 @@ import ProfileHeader from "@/components/People/ProfileHeader.vue";
 import SectionsToDisplay from "@/mixins/SectionsToDisplay.js";
 import StructureDefinition from "@/mixins/StructureDefinition.js";
 import Vue from "vue";
+import MobileLayout from "@/mixins/MobileLayout.js";
 
 export default {
   created(){
     this.screenSize = this.$vuetify.breakpoint.name;
   },
   computed:{
-    fontSizeParagraph(){
-      var fontSize='16px';
-      switch (this.screenSize) {
-        case 'xs': fontSize = '10px'
-          break;
-        case 'sm': fontSize =  '10px'
-          break;
-        case 'md': fontSize =  '14px'
-          break;
-        case 'lg': fontSize =  '16px'
-          break;
-        case 'xl': fontSize =  '16px'
-          break;
-      }
-      return fontSize;
-    },
-    fontSizeH1(){
-      var fontSize = '32px';
-      switch (this.$vuetify.breakpoint.name) {
-        case 'xs': fontSize = '26px';
-          break;
-        case 'sm': fontSize = '26px'
-          break;
-        case 'md': fontSize = '32px'
-          break;
-        case 'lg': fontSize = '32px'
-          break;
-        case 'xl': fontSize = '32px'
-          break;
-      }
-      return fontSize;
-    },
-    gridLayout(){
-      var layout = "";
-      switch (this.screenSize) {
-        case 'xs': layout = 'xs10 pr-3';
-          break;
-        case 'sm': layout = 'xs8 pr-3';
-          break;
-        case 'md': layout = 'xs6 pr-3';
-          break;
-        case 'lg': layout = 'xs6 pr-3';
-          break;
-        case 'xl': layout = 'xs6 pr-3'
-      }
-      return layout;
-    },
-    smallScreenCompute(){
-      var smallScreen=false;
-      if(this.$vuetify.breakpoint.name == "xs")
-      {
-        smallScreen = true;
-      }
-      else{
-        smallScreen = false;
-      }
-      return smallScreen;
+    setGridLayout(){
```

Comment:
Same comment as earlier. You aren't setting anything here.

Reply:
Done